src/frontend/src/lib/services/batch.services.ts

```diff
@@ -0,0 +1,14 @@
+export const batch = async function* <T>({
+	promises,
+	batchSize
```

Comment:
Just `size`? Don't know, today I'm in the mood of shortening any variable names 😂

Reply:
ahahahahah! I noticed! I am just afraid that it is not clear enough, it should say "how many promises per batch"